man/veritytab.xml

```diff
@@ -60,6 +60,13 @@ This is based on crypttab(5).
 
     <variablelist class='fstab-options'>
 
+      <varlistentry>
+        <term><option>hash-offset=<replaceable>BYTES</replaceable></option></term>
+
+        <listitem><para>Offset of hash area/superblock on <literal>hash-device</literal>. (Aligned on 512 bytes.)
```

Comment:
"Must be multiple of 512 bytes"

Reply:
done.

I have reused the string "(Multiples of 512 bytes.)"

Reply:
hum, maybe this should be singular, right?

Reply:
probably singular yes